apps/ingestion/services.py

```
@@ -137,19 +137,19 @@ class RateLimitService:
         self.redis = redis_client
-        self.default_rate_limit = 1000  # events per minute
+        self.default_rate_limit = 100  # events per minute (fallback)
         self.default_window = 60  # seconds
     
     def _get_rate_limit_key(self, tenant_id: str, window: int) -> str:
         """Get Redis key for rate limiting."""
         current_window = int(datetime.utcnow().timestamp() // window)
         return f"rate_limit:{tenant_id}:{current_window}"
     
-    def check_rate_limit(self, tenant_id: str, events_count: int = 1) -> RateLimitInfo:
+    def check_rate_limit(self, tenant_id: str, tenant_rate_limit: int = None, events_count: int = 1) -> RateLimitInfo:
```

Ruff (0.13.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)